modules/exploits/linux/http/vinchin_backup_recovery_cmd_inject.rb

```diff
@@ -0,0 +1,107 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+
+  def initialize(info = {})
+    super(update_info(info,
+        'Name' => 'Vinchin Backup & Recovery Command Injection',
+        'Description' => %q{
+        This module exploits a command injection vulnerability in Vinchin Backup & Recovery
+        versions 5.0 to 7.0. Due to insufficient input validation in the
```

Comment:
The module metadata lists versions up to 7.0, whereas the tested 7.0.1 was also vulnerable.